### ISSUE TITLE
api.hnpwa.com is down

### ISSUE THREAD

**Issue by marcelarie** [pajecawav/better-hn#54]
![satty-20241206-13:28:18](https://github.com/user-attachments/assets/982af70f-65ea-4c01-a583-05f6624a4da5)

The response of the API calls is:
```
Cannot read property 'apply' of undefined
```

**Comment by pajecawav** [comment]
Yeah, hnpwa api goes down sometimes and I think there's nothing I can do about it. The project seems to be unmaintained. I'll see if I can use HN's api directly
